testing/testcafe/runner.js

```diff
@@ -115,3 +132,24 @@ function getArgs() {
         }
     });
 }
+
+function clearTestPage() {
+    return createTestCafe.ClientFunction(() => {
```

Comment:
Why do we reimplement this function?
https://github.com/DevExpress/DevExtreme/pull/23431/files#diff-521e8c2c541410a6f64a734a893d78e2e8b03a83138c34e8d6033b7cb6780a59R3

Reply:
Unfortunately, we cannot import *ts in *js files directly.

The best solution is to compile existing ```clearTestPage``` with ```tsc``` and import it.
But reimplementing this function in js is a fast solution and we agreed with @EugeniyKiyashko that it will be replaced via compiled from ts version in the next PR.

Reply:
Yes it will be leaded to a general view in the future requests